src/components/editor/Status/Saved.tsx

```diff
@@ -0,0 +1,17 @@
+import CloudDoneIcon from '@mui/icons-material/CloudDone';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    iconSize: number;
+}
+
+function Saved({ iconSize }: Props) {
```

Comment:
Broke out the statuses into little component to try to reduce the clutter in the main file.